packages/shared/src/components/alert/AlertContainer.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactElement, ReactNode } from 'react';
+import classed from '../../lib/classed';
+
+interface ClassName {
+  container?: string;
+  overlay?: string;
+}
+
+interface AlertContainerProps {
+  children: ReactNode;
+  className?: ClassName;
+}
+
+export const AlertBackground = classed(
```

Comment:
We can probably extract anything classed to the common file?

Reply:
Good point! Let me work on your concerns about doing some cleaning 🚢